### PR TITLE
add data migration for current languages

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@miltontony @saeedmb8 ready for review

File: molo/core/migrations/0055_migrate_current_site_languages_to_language_relation.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from django.db import migrations, models
+
+
+def convert_languages_to_site_language_relation(apps, schema_editor):
+    from molo.core.models import SiteLanguage, SiteLanguageRelation Main
+    main = Main.objects.all().first()
+    if main:
+        site = main.get_site()
+        if site:
+            language_setting = Languages.objects.create(
```
Comment (miltontony) [inline molo/core/migrations/0055_migrate_current_site_languages_to_language_relation.py]: This should be a `get_or_create`

Comment (codiebeulaine) [inline molo/core/migrations/0055_migrate_current_site_languages_to_language_relation.py]: 👍

**Comment by miltontony** [comment]
👍 wtih

**Pull request merged**